Scuti/Scripts/UI/Store/Offer Details/ZoomOfferDetails.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+namespace Scuti.UI
+{
+    public class ZoomOfferDetails : PanningAndPinchImage, IDragHandler, IBeginDragHandler, IEndDragHandler, IScrollHandler
+    {
+        [Header("Sub Widget")]
+        [SerializeField] private UIDisplayImageZoom displayLargeImage;
+
+        [SerializeField] private bool isDragDetected;
+
+        // ---------------------------------------------------------------------------------------
+
+        void Awake()
+        {
+            displayLargeImage.Init();
+        }
+
+        public void ResetSizeImage()
+        {
+            transform.localScale = initialScale;
+        }
+
+        // ---------------------------------------------------------------------------------------
+
+        /// <summary>
+        /// IScrollHandler: This method is called when scrolling
+        /// </summary>
+        /// <param name="eventData"></param>
+        public void OnScroll(PointerEventData eventData)
+        {
+            base.OnScroll(eventData);
+        }
+
+        /// <summary>
+        /// This method will be called on the start of the mouse drag
+        /// </summary>
+        /// <param name="eventData">mouse pointer event data</param>
+        public void OnBeginDrag(PointerEventData eventData)
+        {
+            base.OnBeginDrag(eventData);
+        }
```

Comment:
@Jpablogs8  These shouldn't be needed.  If they are just calling base there is no reason to make these functions. Or am I missing something?